constants/options.ts

```diff
@@ -0,0 +1,46 @@
+export const GENDERS = ['여성', '남성', '기타'];
```

Comment:
```suggestion
export const GENDERS = ['여성', '남성', '기타'] as const;
```

(p5) 내부를 변경할 수 없도록 `as const` 를 붙이면 좋습니다.